decidim-proposals/app/controllers/decidim/proposals/admin/proposal_answers_controller.rb

```diff
@@ -44,6 +63,42 @@ def skip_manage_component_permission
         def proposal
           @proposal ||= Proposal.where(component: current_component).find(params[:id])
         end
+
+        def proposals
+          @proposals ||= Proposal.where(component: current_component).where(id: params[:proposal_ids])
+        end
+
+        def template
+          @template ||= Decidim::Templates::Template.find(params[:template][:template_id])
+        end
```

Comment:
Here we need to take into account that the `Template` module might not be installed,
we can use this method to check if it is:
```ruby
Decidim.module_installed? :templates
```
If not installed, this option should not appear at all